docs/usage-indicator.md

```diff
@@ -0,0 +1,34 @@
+# Usage Indicator
+
+![alpha](assets/alpha.svg)
+
+Argo Workflows provides an indication of how much resource your workflow has used and save this information. This is intended to be an **indicative but not accurate** value. 
+
+## Calculation
+
+Usage indicator calculation is always an estimate, and is calculated by [../util/usage/estimator.go](../util/usage/estimator.go) based on container duration, specified pod resource requests, limits, or defaults (for memory and CPU). 
+
+Each indicator is divided by a common denominator depending or resource type.
+
+### Example
+
+A pod that runs for 3m, with a CPU limit of 2000m, no memory request and an `nvidia.com/gpu` resource limit of 1:
+
+* CPU: 3 * 60s * 2000m / 1000m = 6m*cpu
+* Memory: 3 * 60s * 100m / 1Gi = 0s*memory
```

Comment:
This makes no sense to me.

What is `m` in this context? Milicores? Mebibytes?

What is a `memory` unit defined as? 

If I take this expression literally, it says to me either "18,000 second-milicores per Gigibyte" or "180 seconds per kibibye", depending on what `m` is here. Neither of these make sense per se or as a measure of memory usage.

How do we end up at "0 second memories" here?